### PR TITLE
Add the openSSH update check to 8.3+

### PR DESCRIPTION
**Pull request by jouyingbin**
**Description:**
OpenSSH 8.3 update will be upgraded to the new version in SLE15 SP3. This PR is going to add the OpenSSH update check to 8.3+ and add the version check and record_info for OpenSSH.

- Related ticket: https://progress.opensuse.org/issues/90458
- Needles: NA
- Verification run: https://openqa.suse.de/tests/5696585#step/openssh_fips/6

### CODE REVIEW

File: tests/fips/openssh/openssh_fips.pm [hunk 1]
```diff
@@ -1,33 +1,46 @@
...
 
+    zypper_call('info openssh');
+    my $current_ver = script_output("rpm -q --qf '%{version}\n' openssh");
+
+    # openssh update to 8.3 in SLE15 SP3
+    if (is_sle('>=15-sp3') && ($current_ver ge 8.3)) {
+        record_info("openssh version", "Current openssh package version: $current_ver");
+    }
+    else {
+        record_info("openssh version outdate", "openssh version need to be updated over to 8.3+ in SLE15 SP3");
```
Comment (rfan1) [inline tests/fips/openssh/openssh_fips.pm:41]: Ben,
I'd suggest to add some "soft fail here", just recording the info will not give any error/warning, we still need check the result manually.

**Review by lilyeyes — APPROVED** [review]
LGTM.

File: tests/fips/openssh/openssh_fips.pm (hunk 1 above)
Comment (jouyingbin) [reply tests/fips/openssh/openssh_fips.pm]: Update the PR. thanks @rfan1

**Review by rfan1 — APPROVED** [review]
LGTM

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**